src/Bundle/CoreBundle/Controller/GraphQLApiController.php

```diff
@@ -45,15 +45,20 @@ public function indexAction(Organization $organization, Domain $domain, Request
             ]
         );
 
-        $server = new StandardServer(
-            ServerConfig::create()->setSchema($schema)->setQueryBatching(true)->setDebug(true)->setContext(
-                function () use ($request) {
-                    return [
-                        'csrf_token' => $request->headers->get('X-CSRF-TOKEN'),
-                    ];
-                }
-            )
+
+        $serverconfig = ServerConfig::create();
+        # set debug flag according to kernel debug
+        $serverconfig->setDebug($this->getParameter('kernel.debug'));
+        $serverconfig->setSchema($schema)->setQueryBatching(true)->setDebug(true)->setContext(
```

Comment:
You are setting the debug flag in line 51 according to kernel.debug parameter but override it in line 52 to true.
You just need to replace setDebug(true) with setDebug($this->getParameter('kernel.debug') and do not need to change any other line